app/public/css/styles.css

```diff
@@ -1577,3 +1532,52 @@ td {
 .flex-align {
   display: flex;
 }
+
+/* ALERTS */
+
+.alert {
+  position: relative;
+  border: 1px solid transparent;
+  padding: 24px 24px 24px 64px;
+  background-color: white;
+  border-radius: 2px;
+}
+
+.alert:before {
+  content: '';
+  position: absolute;
+  left: 24px;
+  width: 24px;
+  height: 24px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+.alert.error {
+  border-color: var(--negative);
```

Comment:
Please, start colors var with `--color-` so, in this case, it would be `var(--color-negative)`. And change the name from *negative* to *error*. It would be `--color-error`.